### PR TITLE
Add option to set team in config for studio

### PR DESCRIPTION
**Pull request by amritghimire**
As part of https://github.com/iterative/studio/issues/10774,
this adds a command as below:

```sh
datachain studio team team_name
Set default team to 'team_name' in <PATH>/.datachain/config
```

```sh
datachain studio team team_name --global
Set default team to 'team_name' in /Users/amritghimire/Library/Application Support/datachain/config
```

### CODE REVIEW

File: src/datachain/cli.py [hunk 1]
```diff
@@ -174,6 +174,30 @@ def add_studio_parser(subparsers, parent_parser) -> None:
         help=studio_logout_help,
     )
 
+    studio_team_help = "Set the default team for Datachain"
```
Comment (shcheklein) [inline src/datachain/cli.py:177]: DataChain (and in other places)

File: src/datachain/cli.py [hunk 2]
```diff
@@ -174,6 +174,30 @@ def add_studio_parser(subparsers, parent_parser) -> None:
...
+        "Set the default team for Datachain to use when interacting with Studio."
+    )
+
+    team_parser = studio_subparser.add_parser(
+        "team",
+        parents=[parent_parser],
+        description=studio_team_description,
+        help=studio_team_help,
+    )
+    # Add a positional argument for the team name
```
Comment (shcheklein) [inline src/datachain/cli.py:188]: do we need this comment?

File: src/datachain/cli.py [hunk 3]
```diff
@@ -174,6 +174,30 @@ def add_studio_parser(subparsers, parent_parser) -> None:
...
+    team_parser.add_argument(
+        "team_name",
+        action="store",
+        help="The name of the team to set as the default.",
+    )
+    team_parser.add_argument(
```
Comment (shcheklein) [inline src/datachain/cli.py:194]: is it system or user config?

**Review by shcheklein — APPROVED** [review]
some minor comments / questions

File: src/datachain/cli.py (hunk 3 above)
Comment (amritghimire) [reply src/datachain/cli.py:193]: By default, it is global config. Example: /Users/amritghimire/Library/Application Support/datachain/config

**Pull request merged**